server/src/index.ts

```diff
@@ -12,6 +12,9 @@ app.get('*', (req: Request, res: Response) => {
   res.send(`Hello ${req.path}`);
 });
 
-app.listen(port, () => {
-  console.log(`Hello server, running on port ${port}`);
-});
+if (require.main === module)
```

Comment:
I would add brackets around this if statement since it's multiline.

Reply:
I have usually omitted curly braces from if statements if there is only one statement after it. Though in this case, it indeed may improve readability.